### PR TITLE
Add react-markdown runtime dep to display it

### PR DESCRIPTION
**Pull request by laem**
Waiting for a build time way to to it, see https://github.com/facebook/docusaurus/discussions/6009

react-markdown adds 133.3 minified (gzipped 44) https://bundlephobia.com/package/react-markdown@7.1.1

### CODE REVIEW

**Comment by mquandalle** [comment]
Thank you! The page including API examples from the previous site is [here](https://github.com/betagouv/publicodes/blob/7a68792e8d39d26c901cc9047a40717a3952ba74/site/pages/M%C3%A9canismes.tsx)

**Review by mquandalle — APPROVED** [review]

**Comment by laem** [comment]
Looks like we have the same runtime problem to create the YAML visualisation of the examples, right ?

**Comment by laem** [comment]
@mquandalle I struggle with Docausorus / MDX. 
My nvim prettier version isn't up to date, and I can't display the arguments, I get obscure react errors :/

File: website/docs/mécanismes.mdx [hunk 1]
```diff
@@ -9,14 +9,23 @@ Docusaurus community https://github.com/facebook/docusaurus/discussions/6009
 -->
 
 import mecanismsData from '../../packages/react-ui/source/data/mecanisms.json'
+import ReactMarkdown from 'react-markdown'
 
 export const MechanismsList = ({ children, color }) => (
 	<>
-		{Object.entries(mecanismsData).map(([mecanismName, meta]) => (
+		{Object.entries(mecanismsData).map(([mecanismName, meta]) => console.log(meta) ||(
```
Comment (mquandalle) [inline website/docs/mécanismes.mdx:16]:
```suggestion
		{Object.entries(mecanismsData).map(([mecanismName, meta]) => (
```

**Pull request merged**